style.css

```diff
@@ -193,7 +193,7 @@ body.lock {
     color: var(--additional-color);
 }
 .header__cartlink {
-    font-size: 12px;
+    font-size: 3.7vw;
```

Comment:
And here 12px is now 3.7 vw so which one is it? Can you please be consistent with your sizes?

Reply:
this is a subjective assessment, it looks better

Reply:
i think if you are using PX across all styling font-size also should be consistent with other

Reply:
Changed all sizes to 3.5 vw to be consistent with each other

Reply:
Went back to px